.travis.yml

```
@@ -52,6 +58,11 @@ before_script:
     - phpunit --version
     - php --version
 
-script: phpunit
+script: 
+  - |
+      if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." ]]; then
```

I think we should put this behind an env var and only run on a single suite (no point running `n` times and slowing all of them down)

```suggestion
      if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." && ${ RUN_PHPCS } == "1" ]]; then
```

And update line 11 to:

```
    - RUN_PHPCS=1 WP_VERSION=latest WP_MULTISITE=0 #Current stable release
```